### PR TITLE
added slack link into discussion section

### PR DESCRIPTION
**Pull request by AlannaHarlton**
Associated Issue: #1826 

### Summary of Changes

* added Slack link to Discussion section or README.md

### Test Plan

I committed it to my fork, then viewed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,9 @@ We strive for collaboration with [mutual respect for each other][contributing].
 
 ### Discussion
 
-We're all on Mozilla's IRC in the [#devtools-html][irc-devtools-html] channel on irc.mozilla.org.
+Our primary communication method is now on the [Slack app][slack]
+
+We are also on Mozilla's IRC in the [#devtools-html][irc-devtools-html] channel on irc.mozilla.org.
```
Comment (jasonLaster) [inline README.md]: I wonder if we can simplify the language a bit and make it consistent with the other options.

```
* **Chat** Say hello in [Slack][slack] or in the [#devtools-html][irc-devtools-html] channel on irc.mozilla.org. 
```

@clarkbw?

File: README.md [hunk 2]
```diff
@@ -43,7 +43,9 @@ We strive for collaboration with [mutual respect for each other][contributing].
 
 ### Discussion
```
Comment (jasonLaster) [inline README.md]: Perhaps we also add the badge to the top:

screen shot 2017-01-24 at 8 21 52 pm

**Comment by clarkbw** [comment]
Woo! screen shot 2017-01-25 at 12 06 52 pm

**Comment by AlannaHarlton** [comment]
Hey, I added the badge in the latest commit and updated the wording. I'm not familiar with markdown, and wanted to use the var for [slack] in the badge but it didn't like that so I just used the full URL. I'm not sure if that is right. Also, it would be nice if all the badges opened in new tabs, but again I'm not sure about markdown. :)

**Comment by Garbee** [comment]
You can't force new window opening through Markdown. It is an abstraction layer to handle *just* the core pieces needed to make HTML (or whatever you want your output to be.) Everything else is handled by the converting engine, which is in Github's control and not ours. So, that's totally fine having it open in the sam window/tab; we don't have a choice in the matter. ;)

**Comment by jasonLaster** [comment]
Yay! First PR! 💃 

Keep em coming @AlannaHarlton!

**Pull request merged**